internal/node/node.go

```diff
@@ -445,11 +445,19 @@ func (n *Node) handleAutostarts() {
 			continue
 		}
 
-		agentDeployRequest := agentWorkloadInfoFromControlDeployRequest(request, autostart.Namespace, numBytes, *workloadHash)
-		agentDeployRequest.TotalBytes = int64(numBytes)
-		agentDeployRequest.Hash = *workloadHash
+		var hash string
+		if workloadHash != nil {
+			hash = *workloadHash // HACK!!! for agent-local workloads, this should be read from the release manifest
```

Comment:
agent-local workloads don't exist(yet)

Reply:
what is a "release manifest" in this context?

Reply:
When we support agent-local workloads, the agent release should include a hash of the expected artifact.